Model/Transaction/Refund.php

```diff
@@ -80,11 +80,13 @@ public function build(
             $adjustmentFee = $creditmemo->getAdjustmentNegative();
             $adjustmentRefund = $creditmemo->getAdjustmentPositive();
 
-            // Discounts on credit memos act as fees and shouldn't be included in $itemDiscounts
-            foreach ($this->request['line_items'] as $k => $lineItem) {
-                $lineItemSubtotal = $lineItem['unit_price'] * $lineItem['quantity'];
-                $this->request['line_items'][$k]['discount'] += ($adjustmentFee * ($lineItemSubtotal / $subtotal));
-                $itemDiscounts += $lineItem['discount'];
+            if ($subtotal != 0) {
+                // Discounts on credit memos act as fees and shouldn't be included in $itemDiscounts
+                foreach ($this->request['line_items'] as $k => $lineItem) {
+                    $lineItemSubtotal = $lineItem['unit_price'] * $lineItem['quantity'];
+                    $this->request['line_items'][$k]['discount'] += ($adjustmentFee * ($lineItemSubtotal / $subtotal));
+                    $itemDiscounts += $lineItem['discount'];
```

Comment:
@ryanreeves-taxjar If a credit memo has a $0 subtotal, do we also want to skip this line on `$itemDiscounts`? This line was initially called prior to modifying discounts based on the adjustment fee. Should we only wrap line 86-87 in the conditional, or keep as-is? Only use case I can imagine this may impact would be a $0 subtotal credit memo with a shipping refund and a shipping discount.